src/commands/publish-helm.yml

```diff
@@ -0,0 +1,38 @@
+description: >
+  Publish a helm chart
+
+parameters:
+  chart-repo:
+    type: string
+    description: Repository for helm chart
+  chart-org:
+    type: string
+    description: Organization for helm chart repo
+
+steps:
+  - run:
+    name: Publish
+    command: |
+      CHART_REPO="<<chart-repo>>"
+      CHART_ORG="<<chart-org>>"
+
+      git config --global user.email $GIT_USER_EMAIL
+      git config --global user.name $GIT_USER
```

Comment:
Can we use parameters for the env variable names? Similar to [this](https://github.com/CircleCI-Public/docker-orb/blob/master/src/commands/check.yml)?

Reply:
Also, since these same lines appear in both publish-helm and tag, can we move them to a different command named configure-git or something and run that first?

Reply:
Idk about environment variables but done